src/bundle/Resources/translations/dashboard.en.yml

```diff
@@ -1,4 +1,27 @@
+# Desc: Everyone
```

Comment:
We currently use crowdin for translation, and it is configured around the xlf format. Do you intend to move to something else ?

Reply:
@bdunogier has a point. What is the plan about the translation topic?
ping @Nattfarinn @sunpietro

Reply:
It should be XLIFF. No idea why there's YML. We'll fix that. :)

Reply:
@webhdx could you address this?